Tests/StackClusterTests.swift

```diff
@@ -161,7 +163,7 @@ class StackClusterTests: QuickSpec {
                     it("should be able to scroll horizontally") {
                         let vc = nestedStack()
                         vc?.loadViewIfNeeded()
-                        
+                        vc?.view.layoutIfNeeded()
```

Comment:
It's probably because you ask to load the view but nothing triggers a layout. You don't need to load the view before doing a snapshot and you don't need to check here if it can scroll since there are already tests for this (if I missed some than add the tests cases but don't mix it with these unrelated specs)